### PR TITLE
Ia qafeedback

### PR DESCRIPTION
**Pull request by ianyimi**
wider premapanel
wider text input and moved panel check beside it
add background to muse onboarding
analytics added to muse hq onboarding finish
normal material on publish button

### CODE REVIEW

File: src/scenes/MuseHQ/components/Onboarding/index.tsx [hunk 1]
```diff
@@ -22,6 +23,7 @@ export default function Onboarding() {
 
   useEffect(() => {
     if (stage === -1) {
+      analytics.capture("hq-onboardingFinish");
```
Comment (alex-shortt) [inline src/scenes/MuseHQ/components/Onboarding/index.tsx:26]: we already have analytics capturing musehq onboarding, the monday item was for tracking the new buildertool onboarding

File: src/tools/BuilderWand/components/PremaPanel/index.tsx [hunk 2]
```diff
@@ -108,41 +112,21 @@ export default function PremaPanel() {
...
         </animated.group>
         <FocusInteractable
-          onHover={() => {setOHover(true)}}
-          onUnHover={() => {setOHover(false)}}
+          onHover={() => {
+            setOHover(true);
+          }}
+          onUnHover={() => {
+            setOHover(false);
+          }}
```
Comment (alex-shortt) [inline src/tools/BuilderWand/components/PremaPanel/index.tsx:129]: one liner

File: src/tools/BuilderWand/components/PremaPanel/index.tsx [hunk 3]
```diff
@@ -155,21 +139,21 @@ export default function PremaPanel() {
               <animated.group position-z={optionsPosZ}>
                 <mesh
                   name="hamburger"
-                  position-x={-0.725*1/HOTBAR_SCALE}
+                  position-x={(-0.725 * 1) / HOTBAR_SCALE}
```
Comment (alex-shortt) [inline src/tools/BuilderWand/components/PremaPanel/index.tsx:142]: *1 is redundant

File: src/tools/BuilderWand/components/PremaPanel/index.tsx [hunk 4]
```diff
@@ -155,21 +139,21 @@ export default function PremaPanel() {
...
                   geometry={nodes.hamburger.geometry}
                 >
                   <meshBasicMaterial color={COLORS.btnPrimary} />
                 </mesh>
               </animated.group>
               <mesh
-                position-x={-0.725*1/HOTBAR_SCALE}
+                position-x={(-0.725 * 1) / HOTBAR_SCALE}
```
Comment (alex-shortt) [inline src/tools/BuilderWand/components/PremaPanel/index.tsx:149]: *1 is redundant

File: src/tools/BuilderWand/components/PremaPanel/index.tsx [hunk 5]
```diff
@@ -155,21 +139,21 @@ export default function PremaPanel() {
...
                 name="hamburger-click"
                 geometry={nodes["props-click"].geometry}
               >
                 <animated.meshBasicMaterial color={optionsColor} />
               </mesh>
               <Text
-                position={[3.55 - (0.725*1/HOTBAR_SCALE), -0.33, 0.075]}
+                position={[3.55 - (0.725 * 1) / HOTBAR_SCALE, -0.33, 0.075]}
```
Comment (alex-shortt) [inline src/tools/BuilderWand/components/PremaPanel/index.tsx:156]: *1 is redundant

File: src/tools/BuilderWand/components/PremaPanel/inputs/TextInput.tsx [hunk 6]
```diff
@@ -11,6 +11,13 @@ import {
   getPropsNodeFromPath,
 } from "modifiers/BuilderMode/utils/tree";
 import { FocusInteractable } from "modifiers/BuilderMode/modifiers/FocusInteractable";
+import { COLORS, FILE_URL, HotbarGLTF } from "../../../assets/constants";
+import { analytics } from "../../../../../utils/analytics";
```
Comment (alex-shortt) [inline src/tools/BuilderWand/components/PremaPanel/inputs/TextInput.tsx:15]: cleaner import `/utils/analytics`

**Review by alex-shortt — CHANGES_REQUESTED** [review]

File: src/tools/BuilderWand/components/PremaPanel/index.tsx (hunk 2 above)
Comment (ianyimi) [reply src/tools/BuilderWand/components/PremaPanel/index.tsx:129]: prettier keeps changing it

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**